pennylane/dla/lie_closure.py

```diff
@@ -163,10 +191,12 @@ def _add_if_independent(M, pauli_sentence, pw_to_idx, rank, num_pw):
 
         Args:
             M (ndarray): coefficient matrix for current LIS
-            pauli_sentence (`~.PauliSentence`): Pauli sentence for which to add a column if independent
+            ps (`~.PauliSentence`): Pauli sentence for which to add a column if independent
```

Comment:
```suggestion
            pauli_sentence (`~.PauliSentence`): Pauli sentence for which to add a column if independent
```
This matches the class signature and the one-line description of the function.

Reply:
✅